bee/drivers/rcc/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_subdirectory(src)
```

Comment:
Maybe select the corresponding source file directory according to the different IC is a better option.